Taarafo.Core/Services/Foundations/GroupPosts/GroupPostService.cs

```diff
@@ -33,5 +35,18 @@ public partial class GroupPostService : IGroupPostService
 
         public IQueryable<GroupPost> RetrieveAllGroupPosts() =>
         TryCatch(() => this.storageBroker.SelectAllGroupPosts());
+
+        public ValueTask<GroupPost> RemoveGroupPostByIdAsync(Guid groupPostId) =>
```

Comment:
Should be `public ValueTask<GroupPost> RemoveGroupPostByIdAsync(Guid groupId, Guid PostId) =>`